src/Services/FHIR/iFhirService.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace OpenEMR\Services\FHIR;
+
+/**
+ * Defines the requirements for FhirService implementations.
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Dixon Whitmire <dixonwh@gmail.com>
+ * @copyright Copyright (c) 2020 Dixon Whitmire <dixonwh@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+interface iFhirService
```

Comment:
Using interface classes in PHP always give me pause. Wouldn't an abstract class be more versatile? Up to you.
@yashrajbothra , you need to be aware here.

Reply:
I'll update this to an abstract class in the next push. Yep for the moment we don't need to define any implementation details or defaults, but agreed that we should just go ahead and make this an abstract class, so that we have the freedom to do so.